### ISSUE TITLE
Go to level page -link does not work if opened in new (background) tab

### ISSUE THREAD

**Issue by ville-j** [elmadev/elmaonline-site#103]
```
ReferenceError: document is not defined

  - Recplayer.js:43 Recplayer.render
    D:/code/elmaonline-site/src/components/Recplayer/Recplayer.js:43:1
```

Seems to be a server side issue in @Ben3eeE code that checks whether the autoplay should be enabled or not.

**Comment by pawq** [comment]
My changes to the level page seem to have fixed this; will confirm when it's pushed to server.

**Comment by pawq** [comment]
Fixed in 9fa7bce51ba890376c212eb73ce9ddac238e9286
